LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2044,6 +2044,54 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
 
                             break;
                         }
+                        case Model.X570_Taichi:
+                        {
+                            v.Add(new Voltage("Vcore", 0, 10, 10));
+                            v.Add(new Voltage("Voltage #2", 1, true));
+                            v.Add(new Voltage("AVCC", 2, 34, 34));
+                            v.Add(new Voltage("+3.3V", 3, 34, 34));
+                            v.Add(new Voltage("Voltage #5", 4, true));
+                            v.Add(new Voltage("Voltage #6", 5, true));
+                            v.Add(new Voltage("Voltage #7", 6, true));
+                            v.Add(new Voltage("3VSB", 7, 34, 34));
+                            v.Add(new Voltage("VBat", 8, 34, 34));
+                            v.Add(new Voltage("VTT", 9));
+                            v.Add(new Voltage("Voltage #11", 10, true));
+                            v.Add(new Voltage("Voltage #12", 11, true));
+                            v.Add(new Voltage("Voltage #13", 12, true));
+                            v.Add(new Voltage("Voltage #14", 13, true));
+                            v.Add(new Voltage("Voltage #15", 14, true));
+                            
+                            //t.Add(new Temperature("CPU Core", 0));
+                            t.Add(new Temperature("Temperature #1", 1));
+                            t.Add(new Temperature("Motherboard", 2));
+                            t.Add(new Temperature("Temperature #3", 3));
+                            //t.Add(new Temperature("Temperature #4", 4));
+                            t.Add(new Temperature("AUXIN2", 5));
+                            t.Add(new Temperature("AUXIN3", 6));
+                            t.Add(new Temperature("AUXIN4", 7));
```

Comment:
Can you name these Auxiliary like the rest? The rest looks good, thanks.

Reply:
I found the `AUXIN*` names in another monitoring software by matching the temperatures.
But I did not find a real world equivalent, so I kept these names.

Similar with `Temperature #1` and `Temperature #3`, I was not able to match the temps to a known sensor or find them in another monitoring software, so I kept the generic names.
Should I rename the `AUXIN*` to the generic `Temperature #` names again?

Reply:
The idea is to keep most of the names somewhat similar, so they're recognizable. This isn't always correct yet, but might as well do it OK now. If they're AUX sensors, you can use Auxiliary 1, Auxiliary 2, etc. Otherwise, Temperature 1/2/3 etc is fine. If nothing is actually read out, it might be better to slash these out. Not all motherboards use all sensors.

Reply:
I did some more searching and testing. These readouts seem to be bogus, probably not connected inputs on the superIO chip.

I removed the unusable ones in my latest commit.